### PR TITLE
Authentication Module On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review auth code module it contains:

- Login & Signup
- Auth Context
- Auth Reducer
- Auth Service ( API Integration )
login-page

sign-up

### CODE REVIEW

File: src/styles/form.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.form-btn{
+    width: 53.6rem !important;
```
Comment (Mohasin-Haque) [inline src/styles/form.css:24]: Why used important?

**Review by Mohasin-Haque — COMMENTED** [review]
Everything looks good to me. You can go ahead.

File: src/styles/form.css (hunk 1 above)
Comment (Rishi05051997) [reply src/styles/form.css:24]: I'm overiding my component lib class

File: src/services/auth-sevice.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            setShowMsg(true);
+        } finally {
+            setShowLoader(false)
+        }
+    }
+
+}
+
+export const createUser = async (firstName, lastName, email, password, userDispatch, setLogin, setShowLoader, setShowMsg, setErrorMsg, navigate) => {
+    debugger;
```
Comment (junaidshaikh-js) [inline src/services/auth-sevice.js]: remove debugger.

**Review by junaidshaikh-js — APPROVED** [review]
Good to merge

File: src/services/auth-sevice.js (hunk 2 above)
Comment (Rishi05051997) [reply src/services/auth-sevice.js]: Yes removed

**Pull request merged**